region/crateria/central/Crateria Super Room.json

```diff
@@ -925,6 +925,14 @@
       ],
       "devNote": "The strat specific requirements are included when clearing the abstract obstacle."
     },
+    {
+      "link": [3, 5],
+      "name": "Remote Acquire Item",
+      "requires": [
+        {"obstaclesCleared": ["C"]}
+      ],
+      "note": "Walk through the item to overload PLMs, fall through the speed blocks, then exit G-Mode to remote acquire the item."
```

Comment:
Using the item to overload PLMs would require the item to be there, but this strat can be run multiple times.  Clearing C requires having Bombs and being in morph, so its still safe.